fluid/neural_machine_translation/transformer/README_cn.md

```diff
@@ -154,9 +154,64 @@ perl multi-bleu.perl data/newstest2013.tok.de < predict.tok.txt
 ```
 BLEU = 25.08, 58.3/31.5/19.6/12.6 (BP=0.966, ratio=0.967, hyp_len=61321, ref_len=63412)
 ```
+###分布式训练
+transformer 模型支持同步或者异步的分布式训练。分布式的配置主要两个方面:
+1 命令行配置
+  --local参数，这个参数有两个取值，True表示单机训练，而False表示使用分布式训练。默认为单机训练模式。
```

Comment:
Please use `` for English names and values in text.

Reply:
ok